cyberark/conjur
# Adds links to NGINX Proxy tutorial in navigation and install

#### What does this pull request do?

It makes it easier to discover the NGINX Proxy tutorial for securing Conjur with TLS.

#### What background context can you provide?

When a person installs Conjur, they should be guided towards eventually securing it with TLS before deploying in production. This PR adds the suggestion to the installation guide and also makes it available in the top navigation.

#### How should this be manually tested?

* Start the docs container as per the README
* Visit the "Install Conjur" page, verify there's a link to the NGINX Proxy tutorial at the bottom
* Verify there's a link under "Tutorials" to NGINX proxy

#### Link to build in Jenkins (if appropriate)

https://jenkins.conjur.net/job/cyberark--conjur/job/tls-tutorial-in-navigation/
